modules/ROOT/pages/create-tokenization-format.adoc

```diff
@@ -1 +1,14 @@
-= Creating the Tokenization Service
+= Creating the Tokenization Format
+
+
+The tokenization format defines the way incoming data is converted. In this example, the domain type used is social security number (SSN). For more information about the supported tokenization formats see xref:tokenization-formats.adoc[Tokenization Formats].
```

Comment:
I know you left off the article because social security number is the name of the domain type, but because it's also a valid English common noun and not capitalized, maybe rewrite to avoid awkwardness? Something like "In this example, social security numbers (SSN) serve as the domain type." ?